Exercise_E.js

```diff
@@ -21,20 +21,30 @@ The output should be always a string with one word, all lowercase.
 Maya won’t forget to thank you at the end of her article :)*/
 
 var KEYS = {
-  "6": "a",
-  "1": "b",
-  "7": "d",
-  "4": "e",
-  "3": "i",
-  "2": "l",
-  "9": "m",
-  "8": "n",
-  "0": "o",
-  "5": "t",
+  6: "a",
+  1: "b",
+  7: "d",
+  4: "e",
+  3: "i",
+  2: "l",
+  9: "m",
+  8: "n",
+  0: "o",
+  5: "t",

```

Comment:
The numbers should be enclosed inside a quotation mark(""), if it represents the key of an object